extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java

```diff
@@ -129,13 +160,18 @@ void registerRestClientListenerForTracing(
     }
 
     @BuildStep
-    @Record(ExecutionTime.STATIC_INIT)
+    @Record(ExecutionTime.RUNTIME_INIT)
```

Comment:
Why this change? Can't we just have stork in its own build step that will use the runtime init phase?

Reply:
I wonder how much we can do at build time. At least having a Stock build item would be good.

Reply:
good catch, I need only Stork initialization moved to runtime

Reply:
@cescoffier what would be the purpose of the stork build item? Mark that initialization is done?

Reply:
@geoand fixed

Reply:
Still trying to see what would be possible. Typically, can we check at build time that the configured discovery and load balancing strategy exist.

Reply:
That would mean that service discovery type has to be build time fixed, no?
I'm not sure we want that